docs/architecture/decisions/0007-react-state-mangement.md

```diff
@@ -0,0 +1,38 @@
+# 7. Fork Cerebral to maintain it as preferred React State Management Solution
+
+Date: 2022-10-06
+
+## Status
+
+Accepted
+
+## Context
+
+During early development, the project decided to use Cerebral to manage state and perform business logic through its sequences of actions within the client application. It has enabled the team to separate business logic that from the display of state.
```

Comment:
```
During early development, the project decided to use Cerebral to manage state and perform business logic through its sequences of actions within the client application. It has enabled the team to separate business logic that from the display of state.
```

Is the `that` in `that from the display of state` supposed to be there?

Reply:
sure isn't.